builder/test-integration/spec/hypriotos-image/docker_spec.rb

```diff
@@ -72,13 +72,6 @@
   it { should be_owned_by 'root' }
 end
 
-describe file('/etc/bash_completion.d/docker') do
```

Comment:
This should test should be remove only until the correct docker-ce package for Debian Stretch can be installed. So maybe we should at least comment this test out and mark it as a todo item.

Reply:
It is cleaner to remove it. When it's part of an upstream package we do not have to test the upstream package in every detail.

Bash completion works for docker cli command with the current DEB package, but I don't know where the bash completion file is.

Reply:
Ok, agreed.

Reply:
Should I remove the tests for docker completely except the first ones for checking the package installation and version? I would only keep theese lines: https://github.com/firecyberice/image-builder-rpi64/blob/add-cloud-init/builder/test-integration/spec/hypriotos-image/docker_spec.rb#L1-L13

Reply:
Not too much changes right now please, we can discuss this later when we redesign the complete build and testing processes.